README.md

```diff
@@ -89,10 +89,29 @@ You can run the linter with:
 npm run lint
 ```
 
-You can run local functional tests with:
+You can run local functional tests:
+
+We use s3cmd and Redis for our functional tests.
```

Comment:
The test suite requires additional tools, **s3cmd** and **Redis** installed in the environment the tests are running in.